pycloudlib/lxd/cloud.py

```diff
@@ -57,19 +92,61 @@ def get_instance(self, instance_id):
         """
         return LXDInstance(instance_id)
 
-    # pylint: disable=R0914
+    def _set_release_image(self, release, is_vm):
```

Comment:
Yes, this one was just to avoid triggering pylint warnings for `R0912`. But since I am disabling it on the `init` method now, there is no need to keep it here